### ISSUE TITLE
pypi installation fails

### ISSUE THREAD

**Issue by enritoomey** [aewallin/allantools#88]
Hi guys, I'm working with python3.4, pip 1.5.6, setuptools 41.0.1, and I'm getting the following error when trying to install allantools:

```
$ python3 --version
Python 3.4.2
$ pip3 --version   
pip 1.5.6 from /usr/lib/python3/dist-packages (python 3.4)
$ python3 -c 'import setuptools;print(setuptools.__version__)'
41.0.1
$ pip3 install allantools
Downloading/unpacking allantools
  Downloading AllanTools-2018.3.tar.gz
  Running setup.py (path:/tmp/pip-build-6u1cwn7x/allantools/setup.py) egg_info for package allantools
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip-build-6u1cwn7x/allantools/setup.py", line 26, in <module>
        long_description=open('README.rst', 'r').read(),
      File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1578: ordinal not in range(128)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip-build-6u1cwn7x/allantools/setup.py", line 26, in <module>

    long_description=open('README.rst', 'r').read(),

  File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode

    return codecs.ascii_decode(input, self.errors)[0]

UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1578: ordinal not in range(128)

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip-build-6u1cwn7x/allantools
Storing debug log for failure in /root/.pip/pip.log
```

So, the workarround that I found is to install directly the code from github, this way:

```
$pip3 install git+http://git@github.com/aewallin/allantools.git
Downloading/unpacking git+http://git@github.com/aewallin/allantools.git
  Cloning http://git@github.com/aewallin/allantools.git to /tmp/pip-7vzv6uo0-build
  Running setup.py (path:/tmp/pip-7vzv6uo0-build/setup.py) egg_info for package from git+http://git@github.com/aewallin/allantools.git
    /tmp/pip-7vzv6uo0-build/setup.py:12: PkgResourcesDeprecationWarning: Parameters to load are deprecated.  Call .resolve and .require separately.
    
Requirement already satisfied (use --upgrade to upgrade): numpy in /usr/lib/python3/dist-packages (from AllanTools==2018.8.1)
Requirement already satisfied (use --upgrade to upgrade): scipy in /usr/lib/python3/dist-packages (from AllanTools==2018.8.1)
Installing collected packages: AllanTools
  Running setup.py install for AllanTools
  Could not find .egg-info directory in install record for AllanTools==2018.8.1 from git+http://git@github.com/aewallin/allantools.git
Successfully installed AllanTools
Cleaning up...
```
I'm not able to update python version, and i'm currently at the latest version of pip and setuptools for python3.4, so to keep python3.4 compatibility, maybe it would be nice if you could release a new pypi version.

**Comment by aewallin** [comment]
Thanks for reporting.

The travis-testing does run Python 3.4 tests also (see tox.ini), but maybe travis does not test "pip install"

**Comment by fmeynadier** [comment]
I think this particular issue has been fixed by commit  db6e719 in the master branch.

Yes, a new pypi release is definitely needed, as it would include this bugfix (among others)  ! I plead guilty for not having done all the stuff I wanted to conribute on the code. Maybe we could have a short review on milestones for next release, also I believe some open issues could now be marked as "solved"... 

@enritoomey, I am not sure to understand if you have troubles because of python3.4 (apart from the "deprecation" warning). Is your installation working correctly now ?

**Comment by enritoomey** [comment]
@fmeynadier  `pip3 install git+http://git@github.com/aewallin/allantools.git`  works perfectly with my python3.4 installation, so I guess its not a python3.4 related issue, and it was solved in master branch, as you mentioned.

**Comment by aewallin** [comment]
there is now a rc1 on pypi, can you try it with
"pip install AllanTools==2019.7rc1"

the final 2019.07 will be tagged next week.

**Comment by aewallin** [comment]
this should be resolved now, since 2019.07 is released.
please reopen or submit a new issue if there still are problems.
closing.
